test/functional/plugin/lsp_spec.lua

```diff
@@ -1380,7 +1380,7 @@ describe('LSP', function()
     end)
   end)
 
-  describe('lsp.util._make_floating_popup_size', function()
+  it('lsp.util._make_floating_popup_size', function()
```

Comment:
I think this should still be `describe`, but was incorrectly nested (my bad!)

Reply:
If this should be a `describe`, then the two `eq`s under it should be wrapped in `it`s in order to run, is that what you had in mind? If so, could you please provide a suggestion as to what the description strings for those `it`s should be?

Reply:
Makes sense; I tried to follow the implicit pattern but obviously missed things.

if each `eq` needs its own `it`, I'd suggest

* `calculates size correctly`
* `calculates size correctly with wrapping`

Reply:
Thanks!

> if each eq needs its own it

Not necessarily, they could also be under the same `it`. They just can't be directly under a `describe` or the tests won't run, it seems.

Having them be under separate `it`s makes it easier to home in on test errors though, so I'll update the PR to that effect and we'll see what maintainers have to say.

Reply:
Cf. 5af9162ee7f828635d54730f8a62c763cfd4b8f0